pkg/cloud/aws/commands/run_test.go

```diff
@@ -1125,6 +1129,7 @@ Summary Report for compliance: my-custom-spec
 	}
 	for _, test := range tests {
 		t.Run(test.name, func(t *testing.T) {
+			clock.SetFakeTime(t, time.Date(2021, 8, 25, 12, 20, 30, 5, time.UTC))
```

Comment:
Just curious why this particular timestamp?

Reply:
@simar7 
It doesn't specify this date intendly. I consider that it just imitates the conventional test code with `clock.SetFakeTime.
It will be fine to be used any date.